packages/application-components/src/components/detail-pages/tabular-detail-page/tabular-detail-page.spec.tsx

```diff
@@ -19,9 +19,7 @@ jest.mock('@commercetools-uikit/utils', () => ({
 const Content = () => (
   <Spacings.Stack scale="m">
     <Switch>
-      <Route exact={true} path="/">
-        <Redirect to="/tab-one" />
-      </Route>
```

Comment:
Nit: this is technically correct.


The docs say 

> Normal `<Redirect>` elements that are not inside a `<Switch>` are ok to remain. 

> Remove any `<Redirect>` elements that are directly inside a `<Switch>`.

So only `<Switch><Redirect /></Switch>` should be migrated.

Using `render` or `children` is also equivalent, although if I remember back then they recommended to prefer `children`.

Reply:
Render `children` is discouraged in v6 as we will need to use the `element` property of the `Route` component so I thought it was easier to follow the prop recommendation.

Reply:
Yes, that makes sense. Thanks!